client/layout/universal-navbar-footer/index.tsx

```diff
@@ -1,10 +1,20 @@
 import './style.scss';
+import { getUrlParts } from '@automattic/calypso-url';
+import { getPathParts, getLanguage } from '@automattic/i18n-utils';
 import { useTranslate } from 'i18n-calypso';
 import SocialLogo from 'calypso/components/social-logo';
 import { navigate } from 'calypso/lib/navigate';
 
+function getRealPathName( path: string ) {
+	const urlParts = getUrlParts( path );
+	const locale = getPathParts( urlParts.pathname ).shift();
+	return 'undefined' === typeof getLanguage( locale ) ? path : path.replace( `${ locale }/`, '' );
```

Comment:
Should we check if there is even a `locale` before passing it to `getLanguage()`?